src/_guides/language/effective-dart/usage.md

```diff
@@ -30,59 +31,58 @@ form&mdash;you do not need to use `+` to concatenate them. Just like in C and
 C++, simply placing them next to each other does it. This is a good way to make
 a single long string that doesn't fit on one line.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (adjacent-strings-literals)"?>
 {% prettify dart %}
 raiseAlarm(
     'ERROR: Parts of the spaceship are on fire. Other '
     'parts are overrun by martians. Unclear which are which.');
 {% endprettify %}
-</div>
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (adjacent-strings-literals)"?>
 {% prettify dart %}
-raiseAlarm(
-    'ERROR: Parts of the spaceship are on fire. Other ' +
+raiseAlarm('ERROR: Parts of the spaceship are on fire. Other ' +
     'parts are overrun by martians. Unclear which are which.');
 {% endprettify %}
-</div>
 
 ### PREFER using interpolation to compose strings and values.
 
 If you're coming from other languages, you're used to using long chains of `+`
 to build a string out of literals and other values. That does work in Dart, but
 it's almost always cleaner and shorter to use interpolation:
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (string-interpolation)"?>
 {% prettify dart %}
 'Hello, $name! You are ${year - birth} years old.';
 {% endprettify %}
-</div>
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (string-interpolation)"?>
 {% prettify dart %}
-'Hello, ' + name + '! You are ' + (year - birth) + ' years old.';
+'Hello, ' + name + '! You are ' + (year - birth) + ' y...';
```

Comment:
I've elided the last string because it's full value isn't essential to this example. Then the full string is given, dartfmt spreads this long String expression onto multiple lines.